bindata/network/ovn-kubernetes/common/ipsec-containerized.yaml

```diff
@@ -143,7 +143,7 @@ spec:
           EOF
             # Wait until the certificate signing request has been signed.
             counter=0
-            until [ -n "$(kubectl get csr -lk8s.ovn.org/ipsec-csr=$(hostname) --sort-by=.metadata.creationTimestamp -o jsonpath='{.items[-1:].status.certificate}' 2>/dev/null)" ]
+            until [ -n "$(kubectl get csr -lk8s.ovn.org/ipsec-csr="$(hostname)" --sort-by=.metadata.creationTimestamp -o jsonpath='{.items[-1:].status.certificate}' 2>/dev/null)" ]
```

Comment:
This doesn't look right, `$(hostname)` is already a part of a quoted command

Reply:
yes, but still shellcheck showed below warning, hence added nested double quote.
```
In bindata/network/ovn-kubernetes/common/ovn-keys-host.sh line 97:
  until [ -n "$(kubectl get csr -lk8s.ovn.org/ipsec-csr=$(hostname) --sort-by=.metadata.creationTimestamp -o jsonpath='{.items[-1:].status.certificate}' 2>/dev/null)" ]
                                                        ^---------^ SC2046: Quote this to prevent word splitting.
```